reV/handlers/sam_resource.py

```diff
@@ -281,13 +317,14 @@ def check_units(var_name, var_array, tech):
                 # convert pressure from Pa to ATM
                 var_array *= 9.86923e-6
 
-        elif 'pressure' in var_name and tech.lower() == 'csp':
-            if np.min(var_array) < 200:
-                # convert pressure from 100 to 1000 hPa
-                var_array *= 10
-            if np.min(var_array) > 80000:
-                # convert pressure from Pa to hPa
-                var_array /= 100
+        elif 'pressure' in var_name:
+            if tech.lower() == 'csp' or tech.lower() == 'troughphysicalheat':
```

Comment:
Why only for trough? Why not SWH and Linear as well? Could make a list of these and do `if tech.lower() in list:`

Reply:
Done